src/Spreadsheet.tsx

```diff
@@ -101,7 +101,10 @@ export type Props<CellType extends Types.CellBase> = {
   /** Callback called on key down inside the spreadsheet. */
   onKeyDown?: (event: React.KeyboardEvent) => void;
   /** Callback called when the Spreadsheet's data changes. */
-  onChange?: (data: Matrix.Matrix<CellType>) => void;
+  onChange?: (
+    data: Matrix.Matrix<CellType>,
+    evaluatedData: Matrix.Matrix<CellType>
```

Comment:
I like this change! Makes it easy to get exactly what you need.
When the API grows, it might be better to expose a single Matrix where each cell has properties like `raw` and `evaluated` along with other data in it. But that's a bit of a higher-level-decision.

Reply:
Yeah, I feel like merging the two data structured together would be better. Should we merge in something separate like this and merge them in together at a later date? Or should we talk about a higher-level change here?

Reply:
For now I think merging them won't be necessary